src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -94,4 +94,14 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// </param>
         ISymbol FindImplementationForInterfaceMember(ISymbol interfaceMember);
     }
+
+    static internal class ITypeSymbolHelpers
+    {
+        // Intentionally not an extension method. We don't want it ever be called for symbol classes
```

Comment:
:question: Why don't we want it called on type classes?

Reply:
> Why don't we want it called on type classes?

Because classes have their own helpers and an interface helper should never be used for them. There is nothing new about this in this change, the helper for an interface got unified.